plugins/modules/swim_intent.py

```diff
@@ -751,7 +763,7 @@ def get_have(self):
                 (site_exists, site_id) = self.site_exists(site_name)
                 if site_exists:
                     have["site_id"] = site_id
-                    self.log("Site Exists: " + str(site_exists) + "\n Site_id:" + str(site_id))
+                    self.log("Site {0} exists having the site id: {1}".format(site_name, str(site_id)), "INFO")
```

Comment:
self.log("The site '{0}' exists and has the site ID '{1}'".format(site_name, str(site_id)), "INFO")

Reply:
updated.